### DISCUSSION TITLE
Regarding IPV6 support for GitHub Enterprise Server

### DISCUSSION THREAD

**Discussion by CJkrishnan** [community/community#101647] [Enterprise]
### Select Topic Area

Question

### Body

Does GitHub Enterprise Edition trail account support the IPV6?
I am actually trying to install GitHub Enterprise Edition trail based on official documentation in GCP with dual_stack configuration.
Installation is successful, I am able to access the IPV4 address but unable to access the IPV6 address.

**Comment by djdefi** [comment]
IPV6 is supported, you may need to manually configure a static IP, or make sure IPV6 is configured with the `ghe-setup-network --visual` command line utility when connected to the GitHub Enterprise Server appliance admin console via SSH.
